stdlib/os/__init__.pyi

```diff
@@ -70,9 +70,20 @@ if sys.platform != "win32":
         POSIX_FADV_WILLNEED: int
         POSIX_FADV_DONTNEED: int
 
-    SF_NODISKIO: int
-    SF_MNOWAIT: int
-    SF_SYNC: int
+    if sys.platform != "linux" and sys.platform != "darwin":
+        # In the os-module docs, these are marked as being available
+        # on "Unix, not Emscripten, not WASI."
```

Comment:
So maybe we should change the os docs?

Can confirm no os.SF_* contants on Linux or macOS.

Reply:
> So maybe we should change the os docs?

Yeah, I think so